vignettes/presize.Rmd

```diff
@@ -0,0 +1,189 @@
+
+### N for a fixed precision
+
+While fixing the sample size is sometimes necessary, it is perhaps more common to select a sample size based on a given precision. For instance, We want to achieve a sensitivity of a given amount (e.g. plus-minus 5%). This is also possible with `presize`. Using the same code as above, we replace the `ntot` argument with the `conf.width` argument. 
+
+```{r}
+(ss <- prec_sens(sens       = .75,   # sensitivity 
+                 prev       = .15,   # prevalence
+                 conf.width = .1,    # CI width
+                 method     = "wilson"))
+```
+
+Under this scenario, `r ceiling(ss$ntot)` participants (of which approximately `r ceiling(ss$n)` would have the condition) will yield a CI width of 0.1 (10%), on average.
+
+Most of the functions in `presize` function the same way, although the sample size argument is generally `n` instead of `ntot` (`prec_sens` and `prec_spec` are special in that you can pass either a number of individuals with a condition to be detected with `n`, or you pass `ntot` and `prev` to get the CI width for a mixed group in which case `n` is derived from `ntot` and `prev`). The calculations for sensitivity can also be preformed if the number of cases rather than the total number and prevalence are available. For instance, if we have 50 individuals with the condition and we expect a sensitivity of 60%, we can put those values in instead.
+
+```{r}
+prec_sens(.6, n = 50, method = "wilson")
+```
+
+Sensitivities and specificities are just proportions so `prec_prop` can also be used for this latter example.
+
+```{r}
+prec_prop(.6, n = 50, method = "wilson")
+```
+
+Using the other functions in `presize` is the same. For instance, the sample size to obtain a CI width of 5 units with a mean of 60 with an SD of 10 is calculated as follows.
+
+```{r}
+prec_mean(22, sd = 10, conf.width = 5)
+```
+
+### Multiple scenarios
+
+It is common for only a vague idea of what to expect in terms of SDs, sensitivities, etc, so it is often worthwhile creating a set of scenarios.
+
+Returning to the colposcope example from above... We want to see how the CI width varies with different sensitivities. With `presize`, it's easy to run different scenarios, simply by passing multiple values to each parameter (where multiple values are passed, they should have the same length!). For instance, for varying a single parameter, scenarios we want to look at can be created with `seq` and passed that to the appropriate `presize` functions argument. Here we vary sensitivity between 50% and 95% in steps of 5%.
+
+```{r}
+(scenario_data <- prec_sens(sens = seq(.5, .95, .05), 
+                            prev = .15, 
+                            ntot = 250, 
+                            method = "wilson"))
+
+```
+
+We can also use `expand.grid` to pass scenarios varying multiple parameters simultaneously. Below we vary sensitivity, prevalence and sample size.
+
+```{r}
+scenarios <- expand.grid(sens = seq(.5, .95, .05),
```

Comment:
Here I would not include so many SENS because we cannot see it in the table, it is the only variation we see in the table and it's the same as above...